### PR TITLE
Tests rendering with integer texture non-zero base level

### PR DESCRIPTION
**Pull request by qkmiao**

### CODE REVIEW

**Comment by qkmiao** [comment]
This test fails in chrome on NVIDIA Windows. I filed chrome bug: https://bugs.chromium.org/p/chromium/issues/detail?id=679639.

**Comment by kenrussell** [comment]
Is this a Chrome bug or does it fail in Firefox in the same configuration? CC'ing @jdashg .

**Comment by qkmiao** [comment]
It's not a chrome bug. It's a NVIDIA Windows driver bug. Firefox also fails on this test.

**Comment by zhenyao** [comment]
Looks good.

File: sdk/tests/conformance2/rendering/draw-with-integer-texture-base-level.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+var canvas;
+var wtu = WebGLTestUtils;
+canvas = document.getElementById("example");
+var gl = wtu.create3DContext(canvas, undefined, 2);
+var tiu = TexImageUtils;
+
+function testDrawIntegerTextureBaseLevel()
+{
+    description("This test verifies the functionality of rendering with integer texture non-zero base level.");
+
```
Comment (zhenyao) [inline sdk/tests/conformance2/rendering/draw-with-integer-texture-base-level.html]: If you have a chromium bug, can you add a comment to point to the bug link? Or provide some details like on which platform it fails, the fashion of the failure.

Comment (qkmiao) [inline sdk/tests/conformance2/rendering/draw-with-integer-texture-base-level.html]: Added failed platforms in comments.

**Comment by zhenyao** [comment]
Thanks.

**Pull request merged**